### ISSUE TITLE
Stereo Normal and Depth Maps Optional

### ISSUE THREAD

**Issue by andreas-at-talon** [colmap/colmap#1200]
Hello,

First of all, great work with colmap. It's incredible and produces great point clouds of great quality.

The problem that I would like to address is more of a time issue. With 169 Images and 4 GPUs, sparse and dense reconstruction takes 2-3 hours minimum to generate the fused.ply. The bottleneck seems to be when generating normal and depth maps. These two folders also take a lot of space when running many many point clouds. 

Since I know when I need to use these .bin files, is there a way to make the writing of the files optional and not have about 10GB per Point Cloud of normal and depth maps written?

Thank you

**Comment by tsattler** [comment]
You can run each step of the reconstruction pipeline individually, both from the command line and the gui. This allows you to only run the stages that you want / need and skip the dense reconstruction stage. You can find more details here: https://colmap.github.io/tutorial.html

**Comment by andreas-at-talon** [comment]
Thank you for your reply,

I want to keep the dense reconstruction stage since it is necessary, I just don't want specifically the two folders specified above. I assume there isn't really a way to say "Don't create these maps".

**Comment by tsattler** [comment]
No, there is no way to not create these folders and the corresponding files. The depth and normal maps are necessary to create the dense point cloud.
